modules/consul/examples_test.go

```diff
@@ -0,0 +1,39 @@
+package consul_test
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/consul"
+)
+
+func ExampleRunContainer() {
+	// runconsulContainer {
+	ctx := context.Background()
+
+	consulContainer, err := consul.RunContainer(ctx,
+		testcontainers.WithImage("docker.io/hashicorp/consul:1.15"),
+	)
+	if err != nil {
+		panic(err)
```

Comment:
In https://github.com/testcontainers/testcontainers-go/pull/2193 we replaced panics with log.Fatalf.

Could you update all of them? 🙏  Please check out the golint exclusions for after-defer exits in that PR